awscli/argparser.py

```diff
@@ -16,6 +16,14 @@
 from difflib import get_close_matches
 
 
+AWS_CLI_V2_MESSAGE = (
+    '\033[93mNote: AWS CLI version 2, the latest major version '
```

Comment:
How portable is this escape code? (And one in the install script as well)

I would prefer we just display it plain, without color and have it show up the same everywhere. Rather than have yellow on some platforms, and a weird escape character on others.

Reply:
I've tried it on Mac, Linux and windows and it worked fine. But I agree there for sure will be some platform where it won't work.